src/PrestaShopBundle/Twig/Extension/LocalizationExtension.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Twig\Extension;
+
+use DateTime;
+use DateTimeInterface;
+use Twig\Extension\AbstractExtension;
+use Twig_SimpleFilter;
+
+class LocalizationExtension extends AbstractExtension
+{
+    /**
+     * @var string
+     */
+    private $dateFormatFull;
+
+    /**
+     * @param string $contextDateFormatFull
+     */
+    public function __construct(string $contextDateFormatFull)
+    {
+        $this->dateFormatFull = $contextDateFormatFull;
+    }
+
+    public function getFilters(): array
+    {
+        return [
+            new Twig_SimpleFilter('date_format_full', [$this, 'dateFormatFull']),
```

Comment:
I can't consider `Y-m-d H:i:s` as a full date format when you can have `2004-02-12T15:19:21+00:00` or `Thu, 21 Dec 2000 16:01:07 +0200`.
Should we have `default_date_format` instead?

Reply:
I don't think so, we have 2 different settings, so date can be displayed in different formats: full & lite.

See https://prnt.sc/pm2fxt